config/routes.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"net/http"
+
+	"github.com/wizelineacademy/golang-bootcamp-2020/domain/repositories"
```

Comment:
By convention, imported packages in a file should be separated into three separated blocks:

1st block: Golang library packages
2nd block: Your project's own dependencies (local packages)
3rd block: 3rd party packages

```suggestion
	"net/http"

	"github.com/wizelineacademy/golang-bootcamp-2020/domain/repositories"
	"github.com/wizelineacademy/golang-bootcamp-2020/interface/handlers"

	"github.com/gorilla/mux"
```